### ISSUE TITLE
Missing new line at EOF with sourcemaps

### ISSUE THREAD

**Issue by marvinhagemeister** [rollup/rollup#756]
When bundling with sourcemaps, rollup doesn't include a line break as the last character. This is in contrast to [how a line is defined in POSIX Standard](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206) which expects a terminating new line characer for each line.

```js
console.log("hello");\n <-- correctly terminated
```
vs:
```js
console.log("hello");\n
//# sourceMappingURL=main.js.map <-- missing \n
```

Note the missing newline character in the second output.

**Comment by Rich-Harris** [comment]
thanks – fixed in 0.34

**Comment by marvinhagemeister** [comment]
Awesome, thanks a lot!
